### PR TITLE
Clean up for 5.0.0 and switch to Bitrise for CI.

### PR DESCRIPTION
**Pull request by ElektrojungeAtWork**
Includes changes from #29

### CODE REVIEW

File: Classes/BITHockeyManager.m [hunk 1]
```diff
@@ -479,21 +479,20 @@ - (void)pingServerForIntegrationStartWorkflowWithTimeString:(NSString *)timeStri
...
   NSURLSessionDataTask *task = [session dataTaskWithRequest:request
-                                          completionHandler: ^(NSData *data, NSURLResponse *response, NSError *error) {
+                                          completionHandler: ^(NSData * __unused data, NSURLResponse *response, NSError * __unused error) {
                                             [session finishTasksAndInvalidate];
                                             
                                             NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse*) response;
                                             [self logPingMessageForStatusCode:httpResponse.statusCode];
                                           }];
-  [task resume];
+  [task resume];
```
Comment (lumaxis) [inline Classes/BITHockeyManager.m]: 🔧 Not sure this whitespace needs to be there 🙂

File: Classes/BITUpdateManager.m [hunk 2]
```diff
@@ -107,52 +107,51 @@ - (void) registerObservers {
...
 }
 
 - (void) unregisterObservers {
-  id strongAppDidEnterBackgroundObserver = self.appDidEnterBackgroundObserver;
-  if(strongAppDidEnterBackgroundObserver) {
-    [[NSNotificationCenter defaultCenter] removeObserver:strongAppDidEnterBackgroundObserver];
+  id strongDidEnterBackgroundObserver = self.appDidEnterBackgroundObserver;
+  id strongDidBecomeActiveObserver = self.appDidBecomeActiveObserver;
+  id strongNetworkDidBecomeReachableObserver = self.networkDidBecomeReachableObserver;
+  if(strongDidEnterBackgroundObserver) {
```
Comment (lumaxis) [inline Classes/BITUpdateManager.m]: 🔧 I think a newline above here would have been nice

File: Classes/BITUpdateManager.m [hunk 3]
```diff
@@ -380,19 +379,19 @@ - (instancetype)init {
     _expiryDate = nil;
     _checkInProgress = NO;
     _dataFound = NO;
-    _updateAvailable = NO;
+    self.updateAvailable = NO;
```
Comment (lumaxis) [inline Classes/BITUpdateManager.m]: ❓ Why are you using the property accessors here?

**Review by lumaxis — APPROVED** [review]

File: Classes/BITUpdateManager.m (hunk 3 above)
Comment (ElektrojungeAtWork) [reply Classes/BITUpdateManager.m]: Thx @lumaxis =)

**Pull request merged**